### ISSUE TITLE
Right-click menu missing options when language mode set to PowerShell but file not PowerShell

### ISSUE THREAD

**Issue by corbob** [PowerShell/vscode-powershell#1473]
### System Details

- Operating system name and version: Windows 10 1803
- VS Code version: Insiders 
- PowerShell extension version: 1.8.2
- Output from `$PSVersionTable`: 

```
Copy / paste the following commands into the PowerShell Integrated Console, and paste the output here:

PS C:\m\r\PT-Tools> code-insiders -v
1.26.0-insider
19df280c7f7cf73d9bafcde82a9e93b14026cfc2
x64
PS C:\m\r\PT-Tools> $pseditor.EditorServicesVersion

Major  Minor  Build  Revision
-----  -----  -----  --------
1      8      2      0


PS C:\m\r\PT-Tools> code-insiders --list-extensions --show-versions
CoenraadS.bracket-pair-colorizer@1.0.59
donjayamanne.githistory@0.4.2
eamodio.gitlens@8.5.4
fatihacet.gitlab-workflow@0.6.0
ms-mssql.mssql@1.4.0
ms-vscode.PowerShell@1.8.2
slevesque.vscode-autohotkey@0.2.2
wesbos.theme-cobalt2@2.1.6
PS C:\m\r\PT-Tools> $PSVersionTable

Name                           Value
----                           -----
PSVersion                      5.1.17134.165
PSEdition                      Desktop
PSCompatibleVersions           {1.0, 2.0, 3.0, 4.0...}
BuildVersion                   10.0.17134.165
CLRVersion                     4.0.30319.42000
WSManStackVersion              3.0
PSRemotingProtocolVersion      2.3
SerializationVersion           1.1.0.1
```

### Issue Description

1. Open an untitled document. 
1. Change language to PowerShell.
1. Select text in the editor
1. Right click
1. * Expected result: Option to lookup help, or run selected text
1. Save file as .ps1 file
1. Right click
1. * now have the option to lookup help, or run selected text

### Attached Logs

I have not attached logs as this appears to be related to lines 161-172 of package.json using resourceLangId instead of editorLangId. Based on https://github.com/Microsoft/vscode/issues/26873#issuecomment-302626728 We should be using editorLangId for this as it's an editor right click menu and doesn't make sense to use it in the Explorer view.

**Comment by tylerl0706** [comment]
Awesome research @corbob - we should change the `when`s to use `editorLangId` instead of `resourceLangId` 😄 

https://github.com/PowerShell/vscode-powershell/blob/master/package.json#L163-L170

**Comment by corbob** [comment]
PR #1475 opened for this 😄

**Comment by rjmholt** [comment]
Closed by #1475

**Comment by rjmholt** [comment]
Closed by #1475
